### PR TITLE
Add home and appVersion key for minecraft

### PR DESCRIPTION
**Pull request by AdamDang**
The key "appVersion"and "home" are needed for ci testing, they are missing in this yaml. That sometimes will cause testing failure.

### CODE REVIEW

File: stable/minecraft/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 name: minecraft
-version: 0.2.0
+version: 0.2.1
+appVersion: latest
```
Comment (timstoop) [inline stable/minecraft/Chart.yaml]: I don't think this is a valid value for appVersion. I'd recommend simply checking what the current latest is and using that value.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/minecraft/Chart.yaml (hunk 1 above)
Comment (AdamDang) [reply stable/minecraft/Chart.yaml]: In the values.yaml: 
 # One of: LATEST, SNAPSHOT, or a specific version (ie: "1.7.9").
  version: "LATEST"
So I think we can use 1.7.9

File: stable/minecraft/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 name: minecraft
-version: 0.2.0
+version: 0.2.1
+appVersion: 1.7.9
```
Comment (timstoop) [inline stable/minecraft/Chart.yaml]: Not sure how you found 1.7.9, but when I check the image (and it's a pretty rabbit hole), I get to 1.12.2 as being the latest. I recommend to use that, as it is at least correct for the time being.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/minecraft/Chart.yaml (hunk 2 above)
Comment (AdamDang) [reply stable/minecraft/Chart.yaml]: Thanks @timstoop 
So we use 1.12.2 for that. It's modified.

**Pull request merged**